### PR TITLE
Add Abhipsha to contributors

### PR DESCRIPTION
**Pull request by chiral-carbon**

### CODE REVIEW

**Comment by knrt10** [comment]
cc @p-society/2021-batch review this PR.

File: _contributors-app/chiral-carbon.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Abhipsha Das
+description: Machine Learning enthusiast.
+picUrl: https:https://avatars2.githubusercontent.com/u/25850628?s=400&v=4
```
Comment (sudonitesh) [inline _contributors-app/chiral-carbon.md]: picUrl  is wrong.

Comment (sudonitesh) [inline _contributors-app/chiral-carbon.md]: picUrl is wrong.

**Comment by chiral-carbon** [comment]
Oops. Correcting it.

**Comment by knrt10** [comment]
Remember, don’t push new commit, ammend it

**Comment by sibasish14** [comment]
Looks good to me👍

**Comment by itsmepiyush2** [comment]
Looks fine to me too.

**Pull request merged**